modules/nf-core/bedtools/multiinter/main.nf

```diff
@@ -0,0 +1,36 @@
+process BEDTOOLS_MULTIINTER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::bedtools=2.30.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bedtools:2.30.0--hc088bd4_0' :
+        'quay.io/biocontainers/bedtools:2.30.0--hc088bd4_0' }"
+
+    input:
+    tuple val(meta), path(beds, stageAs: "inputs/*")
```

Comment:
before add ing stageAs everywhere, can we have abigger discussion about this in #modules please?

Reply:
I've got a good reason this time, I got actual complaints from the tests about name collisions

Reply:
Not AFAIK, but the modified input location has been known to show up in reports

Reply:
Friederike? Any more comments?

Reply:
a different prefix would fix it, but I find this more elegant as it doesn't require any input from the user and "just works"